src/plugins/visualizations/kibana.jsonc

```diff
@@ -45,7 +45,8 @@
       "kibanaUtils",
       "kibanaReact",
       "charts",
-      "savedObjects"
+      "savedObjects",
+      "presentationPanel"
```

Comment:
Why is this bundle necessary?

Reply:
`ErrorEmbeddable` is moved from embeddable plugin to visualizations plugin and  src/plugins/visualizations/public/legacy/embeddable/error_embeddable.tsx requires `PresentationPanelError`.